lib/src/formats/formats.dart

```diff
@@ -293,3 +302,9 @@ Image decodePsd(List<int> bytes) {
 Image decodeExr(List<int> bytes, {double exposure = 1.0}) {
   return ExrDecoder(exposure: exposure).decodeImage(bytes);
 }
+
+/// Decode an OpenEXR formatted image, tone-mapped using the
+/// given [exposure] to a low-dynamic-range [Image].
+Image decodeBmp(List<int> bytes, {double exposure = 1.0}) {
```

Comment:
decodeBmp doesn't need the exposure argument.

Reply:
My apologies thought I removed that, fixed in https://github.com/brendan-duncan/image/pull/164/commits/8d45c144cd56c0fa4b428db0d119e249bf9eb1a6